LocationManager/INTULocationManager/INTULocationManager.m

```diff
@@ -709,7 +709,7 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
     INTULMLog(@"Location services error: %@", [error localizedDescription]);
     self.updateFailed = YES;
     
-    [self completeAllLocationRequests];
+    [self processLocationRequests];
```

Comment:
I agree that this provides better behavior for recurring requests.

However, for non-recurring ones (one-time location requests), this is a relatively significant change in behavior. Previously the idea was that an error would "complete" any outstanding one-time location request, effectively a failure. You'd have to start a new request to try again. With this change, now a one-time location request may have its handler block called multiple times (it will be called for each error, as well as when the request times out or when the request succeeds).

I think either of the above ways of doing things are reasonable, but in order to avoid such a significant change to how the library has worked previously, I would recommend retaining the same behavior of completing all one-time requests when an error comes in. This means just allowing subscriptions to remain active if there is an error.

(We'll need to update the README on this as well.)